nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -378,7 +383,7 @@ def list_available_models(cls) -> List[PretrainedModelInfo]:
 
         model = PretrainedModelInfo(
             pretrained_model_name="stt_en_citrinet_256_gamma_0_25",
-            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_en_citrinet_256_gamma_0_25",
+            description="For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:\nemo:stt_en_citrinet_256_gamma_0_25",
```

Comment:
Added \ ?

Reply:
Yeah, this is to supress lint warnings. I didn;t do it for links but just for description.